### PR TITLE
Add more CYBER

### PR DESCRIPTION
**Pull request by The-Compiler**
```
<Melsie> Der flamebot soll gefälligst per CYBER getriggert werden
```

I can't agree more! 😆 

cc @fmorgner because he's clearly HSR's [residential cyber expert](https://github.com/fmorgner/dev-cyber).

### CODE REVIEW

**Comment by dbrgn** [comment]
Haha, great.

Is there a way to preview those stickers?

**Comment by The-Compiler** [comment]
I don't know, but have a screenshot :wink:

image

**Review by dbrgn — APPROVED** [review]

File: FlameBot.js [hunk 1]
```diff
@@ -110,6 +110,18 @@ class FlameBot {
...
+      if (/cyber/i.test(message.text)) {
+          let sticker;
+          if (Math.random() < 0.5) {
+              // CYBER ATTACKS AHEAD
+              sticker = new Sticker('CAADAgADnAADGW8XB9XpWOhkV96rAg');
+          } else {
+              // CYBER
+              sticker = new Sticker('CAADAgADlgADGW8XB1uhplTaDTPOAg');
+          }
+          this.reply(sticker, message);
```
Comment (rnestler) [inline FlameBot.js:122]: Shouldn't we `return` here? Otherwise one gets a cyber sticker and maybe a random response.

Comment (The-Compiler) [reply FlameBot.js:122]: True, good catch! Fixed.

**Comment by The-Compiler** [comment]
ping? :wink:

**Pull request merged**